app/main/controllers/template/rtmedia-functions.php

```diff
@@ -771,6 +756,24 @@ function rtmedia_image( $size = 'rt_media_thumbnail', $id = false, $echo = true,
 
 }
 
+/**
+ * Appends timestamp at the end of the URL to bypass cache and load fresh image.
+ *
+ * @param string $src Image URL.
+ *
+ * @return string Modified URL.
+ */
+function rtmedia_append_timestamp_in_url( $src ) {
+	$src_query = wp_parse_url( $src, PHP_URL_QUERY );
+	if ( empty( $src_query ) ) {
+		$src .= '?' . time();
+	} elseif ( false === strpos( $src, 'amazonaws.com' ) ) {
```

Comment:
Just for information: Why we've skipped amazonaws.com

Reply:
https://github.com/rtCamp/rtMedia/pull/1664#issuecomment-650266868